packages/hardhat-plugin/src/type-extensions.ts

```diff
@@ -14,10 +14,14 @@ declare module "hardhat/types/config" {
   }
 
   export interface HardhatUserConfig {
-    ignition?: Partial<DeployConfig>;
+    ignition?: Partial<DeployConfig> & {
+      strategyConfig?: Partial<StrategyConfig>;
```

Comment:
My mental model here is that because the `deploy` function of core expects the Strategy object to be newed up, the config will already have been amalgamated into the Strategy before we pass it on. Hence `DeployConfig` will never have StrategyConfig.

Reply:
I don't understand the newed up part.